sklearn/semi_supervised/_self_training.py

```diff
@@ -7,7 +7,7 @@
 from sklearn.base import ClassifierMixin
 
 from ..base import BaseEstimator, MetaEstimatorMixin, _fit_context, clone
-from ..utils import Bunch, safe_mask
+from ..utils import Bunch, get_tags, safe_mask
```

Comment:
Oh, could you modify the import above and import `ClassifierMixin` from the relative import `..base`.